gz_ros2_control/CMakeLists.txt

```diff
@@ -24,39 +24,41 @@ find_package(pluginlib REQUIRED)
 find_package(rclcpp REQUIRED)
 find_package(yaml_cpp_vendor REQUIRED)
 
-if("$ENV{IGNITION_VERSION}" STREQUAL "citadel")
-  find_package(ignition-gazebo3 REQUIRED)
-  set(IGN_GAZEBO_VER ${ignition-gazebo3_VERSION_MAJOR})
-  message(STATUS "Compiling against Ignition Citadel")
-
-elseif("$ENV{IGNITION_VERSION}" STREQUAL "edifice")
-  find_package(ignition-gazebo5 REQUIRED)
-  set(IGN_GAZEBO_VER ${ignition-gazebo5_VERSION_MAJOR})
-  message(STATUS "Compiling against Ignition Edifice")
-
-elseif("$ENV{IGNITION_VERSION}" STREQUAL "fortress")
+if("$ENV{GZ_VERSION}" STREQUAL "fortress")
   find_package(ignition-gazebo6 REQUIRED)
-  set(IGN_GAZEBO_VER ${ignition-gazebo6_VERSION_MAJOR})
-  message(STATUS "Compiling against Ignition Fortress")
+  set(GZ_SIM_VER ${ignition-gazebo6_VERSION_MAJOR})
+  message(STATUS "Compiling against Gazebo Fortress")
 
 else()
-  find_package(ignition-gazebo6 REQUIRED)
-  set(IGN_GAZEBO_VER ${ignition-gazebo6_VERSION_MAJOR})
-  message(STATUS "Compiling against Ignition Fortress")
+  find_package(gz-sim7 REQUIRED)
+  set(GZ_SIM_VER ${gz-sim7_VERSION_MAJOR})
+  message(STATUS "Compiling against Gazebo Garden")
 endif()
 
-find_package(ignition-plugin1 REQUIRED)
-set(IGN_PLUGIN_VER ${ignition-plugin1_VERSION_MAJOR})
+SET(GZ_PLUGIN)
+SET(GZ_SIM)
+if("$ENV{GZ_VERSION}" STREQUAL "fortress")
```

Comment:
Why not do this in the if above?

Reply:
https://github.com/ros-controls/gz_ros2_control/pull/67/commits/1f011b322ff1d36a9eb0a539bb53b9b778aa3c82